polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java

```diff
@@ -1320,7 +1320,7 @@ private void validateMetadataFileInTableDir(
     return storageInfoEntity;
   }
 
-  private class BasePolarisViewOperations extends BaseViewOperations {
+  protected class BasePolarisViewOperations extends BaseViewOperations {
```

Comment:
Why this change?

Reply:
It was violating the visibility of [BasePolarisViewOperations](https://github.com/polaris-catalog/polaris/blob/5c78eefe0f029b5140d41f74c3c4fc1710fc5e25/polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java#L707) .